osm2gtfs/core/osm_connector.py

```diff
@@ -448,90 +448,88 @@ def _build_station(self, stop_area, osm_type):
+        # Analzyse member objects (stops) of this stop area
+        members = {}
+        for member in stop_area.members:
 
-                if member.role == "platform":
+            if member.role == "platform":
 
-                    if isinstance(member, overpy.RelationNode):
-                        member_osm_type = "node"
-                    elif isinstance(member, overpy.RelationWay):
-                        member_osm_type = "way"
+                if isinstance(member, overpy.RelationNode):
+                    member_osm_type = "node"
+                elif isinstance(member, overpy.RelationWay):
+                    member_osm_type = "way"
 
-                    identifier = member_osm_type + "/" + str(member.ref)
+                identifier = member_osm_type + "/" + str(member.ref)
 
-                    if identifier in self.stops['regular']:
+                if identifier in self.stops['regular']:
 
-                        # Collect the Stop objects that are members
-                        # of this Station
-                        members[identifier] = self.stops['regular'][identifier]
-                    else:
-                        sys.stderr.write(
-                            "Error: Station member was not found in data")
-                        sys.stderr.write("https://osm.org/relation/" +
-                                         str(stop_area.id) + "\n")
-                        sys.stderr.write("https://osm.org/node/" +
-                                         str(member.ref) + "\n")
-            if len(members) < 1:
-                # Stop areas with only one stop, are not stations they just
-                # group different elements of one stop together.
-                sys.stderr.write(
-                    "Error: Station with no members has been discarted:\n")
-                sys.stderr.write("https://osm.org/relation/" +
-                                 str(stop_area.id) + "\n")
-                return False
+                    # Collect the Stop objects that are members
+                    # of this Station
+                    members[identifier] = self.stops['regular'][identifier]
+                else:
+                    sys.stderr.write(
+                        "Error: Station member was not found in data")
+                    sys.stderr.write("https://osm.org/relation/" +
+                                     str(stop_area.id) + "\n")
+                    sys.stderr.write("https://osm.org/node/" +
+                                     str(member.ref) + "\n")
+        if len(members) < 1:
+            # Stop areas with only one stop, are not stations they just
+            # group different elements of one stop together.
+            sys.stderr.write(
+                "Error: Station with no members has been discarted:\n")
+            sys.stderr.write("https://osm.org/relation/" +
+                             str(stop_area.id) + "\n")
+            return False
```

Comment:
This method is sometimes returning `None` and sometimes `False`. This should be unified.